DOCS: Update documentation for MonoBehaviour OnMouse events (ISX-2383) (#2301)
Unity-Technologies/InputSystem: Packages/com.unity.inputsystem/CHANGELOG.md

## [Unreleased] - yyyy-mm-dd

### Changed

> - Adjusted documentation to reflect that the OnMouse MonoBehaviour events are supported starting from Unity 6.4 onwards.

Reviewer: ```suggestion
- Updated documentation to reflect that the OnMouse MonoBehaviour events are now supported in Unity 6.4 and above.
```
